mod-source-record-storage-server/src/main/java/org/folio/dao/RecordDaoImpl.java

```diff
@@ -175,6 +177,32 @@
     });
   }
 
+  @Override
+  public Future<Optional<SourceRecord>> getSourceRecord(String id, ExternalIdType externalIdType, String tenantId) {
```

Comment:
wouldn't it be more convenient to have separate dao methods to retrieve sourceRecord by id (aka matchedId) and by externalId of any kind?

Reply:
Sure, will do it.